scripts/ci/selective_tests.sh

```diff
@@ -125,18 +182,60 @@ function count_changed_files() {
     echo "${count_changed_files}"
 }
 
-function run_all_tests_when_push_or_schedule() {
+function run_everything_when_push_or_schedule() {
     if [[ ${GITHUB_EVENT_NAME} == "push" || ${GITHUB_EVENT_NAME} == "schedule" ]]; then
         echo
         echo "Always run all tests in case of push/schedule events"
         echo
-        set_outputs_run_all_tests
-        exit
+        set_outputs_run_everything_and_exit
+    fi
+}
+
+function check_if_api_tests_should_be_run() {
+    local api_triggering_patterns=(
+        "^airflow/api"
+    )
+    pattern_array=("${api_triggering_patterns[@]}")
```

Comment:
Why have `api_triggering_patterns` local var at all?

```suggestion
    pattern_array=(
        "^airflow/api"
    )
```

Reply:
Yeah I think it was a remnant of the earlier version and we do not need it at all any more. those variables were used elsewhere before but indeed they can be removed completely - you are right. Will do.